dash/backend/src/modules/policy/services/policy-service.ts

```diff
@@ -93,5 +94,15 @@ export class PolicyService {
         return await this.auditLogService.calculateMetaData(previous, updated, 'Policy');
     }
 
+    validateActivePolicyScanners(policy: PolicyDto, scanners: ScannerDto[]): boolean {
+        // Checks if an active policy has at least one active scanner
+        if (policy.enabled) {
```

Comment:
Might be able to simplify using https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some

Reply:
return !policy.enabled || scanners.some(...)

Reply:
return !policy.enabled || scanners.some(scanner?.enabled);

Reply:
Haven't run into that function before, thanks